src/__test__/components/plots/styling/SingleGeneSelection.test.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import _ from 'lodash';
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+
+import fakeConfig from '__test__/data/componentConfigs/embeddingContinuous.json';
+
+import SingleGeneSelection from 'components/plots/styling/SingleGeneSelection';
+
+const loadAndRenderSingleGeneSelection = async (config, setSearchedGene) => {
+  await act(async () => {
```

Comment:
Why do you need all this act wraps everywhere, when I write tests I dont use them and I don't get errors?

Reply:
I thought they were needed for the enzyme tests

Reply:
I think it is a react testing thing, when you perform actions that should cause re renderings wrapping these actions in act ensures the updates are done. https://reactjs.org/docs/testing-recipes.html#act